p2p/net/upgrader/upgrader_test.go

```diff
@@ -77,12 +95,18 @@ func (m *blockingMuxer) Unblock() {
 // errorMuxer is a muxer that errors while setting up
 type errorMuxer struct{}
 
-var _ network.Multiplexer = &errorMuxer{}
+var _ upgrader.MsTransport = &errorMuxer{}
 
-func (m *errorMuxer) NewConn(c net.Conn, isServer bool, scope network.PeerScope) (network.MuxedConn, error) {
+func (m *errorMuxer) NegotiateMuxer(c net.Conn, isServer bool) (*upgrader.Multiplexer, error) {
```

Comment:
This is a different test now. The old test errored when creating a muxed connection. This now errors when negotiation. I don't think we should change this test, but it would maybe we should add another test if it makes sense.

Reply:
As the multistream transport itself is removed, this test is no longer relevant so I think it should be removed now.